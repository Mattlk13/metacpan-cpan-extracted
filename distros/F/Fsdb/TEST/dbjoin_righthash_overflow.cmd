prog='dbjoin'
args='-m righthash --input TEST/dbjoin_ex.in --input TEST/dbjoin_ex_overflow.in-2 -n cid'
in=/dev/null
cmp='diff -c -b '
