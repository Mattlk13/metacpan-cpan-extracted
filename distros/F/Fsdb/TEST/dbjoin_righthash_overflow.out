#fsdb cid sid cname
10	1	overflow-once-0
10	1	overflow-once-1
11	2	overflow-twice-0
11	2	overflow-twice-1
11	2	overflow-twice-2
12	1	non-overflow
12	2	non-overflow
#  | dbsort -n cid
#   | dbjoin -m righthash --input TEST/dbjoin_ex.in --input TEST/dbjoin_ex_overflow.in-2 -n cid
